test/cypress/integration/widgets/blueprint_catalog_spec.ts

```diff
@@ -58,4 +58,25 @@ describe('Blueprints catalog widget', () => {
             cy.contains('Description').should('not.exist');
         });
     });
+
+    it('should have segment with correct icons', () => {
+        cy.intercept('/console/external/content*', { fixture: 'blueprints/blueprintsCatalog.json' }).as(
+            'blueprintsCatalog'
+        );
+        cy.usePageMock('blueprintCatalog', {
+            jsonPath: 'test', // this is required to avoid using default mock
+            displayStyle: 'table',
+            fieldsToShow: ['Name', 'Description']
+        }).mockLogin();
+        cy.wait('@blueprintsCatalog');
+        const iconNames = ['gitlab', 'bitbucket', 'git'];
+
+        iconNames.forEach(iconName => {
+            cy.get('.blueprintCatalogWidget').within(() => {
+                cy.get(`tr[class*="${iconName}"]`).within(() => {
+                    cy.get(`i.${iconName}.icon`).should('have.class', iconName);
```

Comment:
Actually `cy.get(`i.${iconName}.icon`)` guarantees that we have class `iconName` 🙂 

So maybe something like:
```suggestion
                    cy.get(`.icon`).should('have.class', iconName);
```
or better would be if we can use `title` attribute to get proper icon element?

Reply:
Yep, looks better 👍🏻